### ISSUE TITLE
Text isn’t read properly with keyboard navigation

### ISSUE THREAD

**Issue by rjayapra** [microsoft/botframework-sdk#6663]
## This is not for asking questions or filing bugs

To get help with an issue with your bot
* ["How To"](https://stackoverflow.com/help/how-to-ask) questions should be submitted on [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework).

To file an issue against a component please go to the components repo
* [Create Azure Bot Framework Service or Channel issue](https://github.com/Microsoft/botframework-services/issues/new/choose)
* [Create Bot Builder Tools issue](https://github.com/Microsoft/botbuilder-tools/issues/new/choose)
* [Create Bot Builder SDK V3 (.NET or Javascript) issue](https://github.com/Microsoft/botbuilder-V3/issues/new/choose)
* [Create Bot Builder SDK V4 .NET issue](https://github.com/Microsoft/botbuilder-dotnet/issues/new/choose)
* [Create Bot Builder SDK V4 Javascript issue](https://github.com/Microsoft/botbuilder-js/issues/new/choose)
* [Create Bot Builder SDK V4 Java issue](https://github.com/Microsoft/botbuilder-java/issues/new/choose)
* [Create Bot Builder SDK V4 Python issue](https://github.com/Microsoft/botbuilder-python/issues/new/choose)

## Issue

When we enable the 'Read Text' option in the chat widget,and the keyboard is used to focus on adaptive card containing text and buttons, the text isnt read out by the screen reader.When tabbing through multiple buttons, visual focus remains on the first button. It happens only when the screen reader is on.

## Proposed change

*Describe the proposed solution*

## Component Impact 

*Describe which components need to be updated*

## Customer Impact 

High Priority. Customer is unable to release the product because of accessibility failure.

**Comment by tracyboehrer** [comment]
WebChat perhaps?
